src/ddscxx/include/org/eclipse/cyclonedds/pub/qos/DataWriterQosDelegate.hpp

```diff
@@ -88,6 +89,7 @@ class OMG_DDS_API DataWriterQosDelegate
     dds::core::policy::OwnershipStrength       strength_;
 #endif  // OMG_DDS_OWNERSHIP_SUPPORT
     dds::core::policy::WriterDataLifecycle     lifecycle_;
+    dds::core::policy::DataRepresentation      datarepresentation_;
```

Comment:
Please also encapsulate this line with the conditional OMG_DDS_EXTENSIBLE_AND_DYNAMIC_TOPIC_TYPE_SUPPORT macro.